docs/mecsimcalc-library/plotdraw.md

```diff
@@ -0,0 +1,1108 @@
+) -> tuple
+```
+
+#### Description:
+
+Plots a line segment between two points and adds a label at the end point.
+
+#### Arguments:
+
+| Argument                 | Type                 | Description                                                         |
+| ------------------------ | -------------------- | ------------------------------------------------------------------- |
+| **`start_point_pixels`** | **tuple**            | The starting point of the line segment (x, y).                      |
+| **`end_point_pixels`**   | **tuple**            | The ending point of the line segment (x, y).                        |
+| **`line_properties`**    | **dict** (optional)  | Properties for the line, including color, linewidth, and linestyle. |
+| **`text`**               | **str** (optional)   | The text to display near the end point of the line segment.         |
+| **`min_spacing`**        | **float** (optional) | Minimum spacing for the text from the end point.                    |
+| **`fontsize`**           | **int** (optional)   | Font size of the text.                                              |
+| **`text_loc`**           | **dict** (optional)  | Dictionary specifying horizontal and vertical alignment of          |
+
+the text. |
+| **`alpha`** | **float** (optional) | Transparency level of the line segment. |
+
+#### Example:
+
+```python
+>>> import matplotlib.pyplot as plt
+>>> start = (100, 200)
+>>> end = (400, 500)
+>>> plot_segment_pixels(start, end, text="Segment", min_spacing=50)
+>>> plt.show()
+```
+
+### plot_annotate_arrow
+
+[**[Source]**](https://github.com/MecSimCalc/MecSimCalc-utils/blob/v0.1.8/mecsimcalc/eng130/plot_draw.py#L551C1-L699C1)
+
+```python
+plot_annotate_arrow(
+    start_point: tuple,
+    trig_angle: float,
+    vec_length: float,
+    text: str = "",
+    min_spacing: float = 150,
+    fontsize: int = 11,
+    text_loc: dict = {"ha": "center", "va": "top"},
+    arrow_properties: dict = {
+        "width": 2,
+        "head_width": 15,
+        "head_length": 15,
+        "fc": "black",
+        "ec": "black"
+    },
+    reverse_arrow: str = "no",
+    text_in_center: str = "no",
+    rev_text: str = "no",
+    alpha: float = 0.8
+) -> tuple
+```
+
+#### Description:
+
+Plots an annotated arrow starting from a given point and extending in a given direction.
+
+#### Arguments:
+
+| Argument               | Type                 | Description                                                                                               |
+| ---------------------- | -------------------- | --------------------------------------------------------------------------------------------------------- |
+| **`start_point`**      | **tuple**            | The starting point of the arrow (x, y).                                                                   |
+| **`trig_angle`**       | **float**            | The angle of the arrow in degrees.                                                                        |
```

Comment:
Why do some use radians and others degrees?

Reply:
No clue, ill be sure to modify it in the next version so all of them include both